src/components/buttons/edit-button.js

```diff
@@ -2,8 +2,10 @@ import React from 'react';
 
```

Comment:
Please, put `edit-button.js` inside its own folder `edit-button` and convert it to normal component folder

Reply:
Done!